home/home_2.1/HelloWorld.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <unordered_map>
+#include <sstream>
+
+const std::string FILENAME = "user_stats.txt";
```

Comment:
constexpr auto should work. also it is better to wrap local constants to anonymous namespace to restrict their visibility to this file only